src/modal/modal.ts

```diff
@@ -21,7 +21,7 @@ export interface NgbModalOptions {
   /**
    * Size of a new modal window.
    */
-  size?: 'sm' | 'lg';
+  size?: 'sm' | 'lg' | 'md';
```

Comment:
then it would be goot if you could extend the api to tell that the default value is not "sm" or "lg" that the default value is "md"